assets/helpers/utilities.js

```diff
@@ -42,3 +42,16 @@ export function clickSubstituteKeyPressHandler(handler?: () => void = () => {})
   };
 }
 
+// Attempts to parse a boolean from a string.
+export function parseBoolean(boolString: string, fallback: boolean): boolean {
+
+  switch (boolString) {
```

Comment:
I would do: boolString.toLowerCase(). Therefore `'True'` and `'true'` will be `true`. Does the switch do a `==` or `===` comparison?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#Description